cps/src/main/java/org/jenkinsci/plugins/workflow/cps/Snippetizer.java

```diff
@@ -231,103 +228,16 @@ public HttpResponse doGenerateSnippet(StaplerRequest req, @QueryParameter String
     }
 
     @Restricted(NoExternalUse.class)
-    public static final String STATIC_URL = ACTION_URL + "/static";
+    public static final String GDSL_URL = ACTION_URL + "/gdsl";
```

Comment:
If you are switching to Groovy views, you can get rid of these constants, inlining their code. I defined `STATIC_URL` for the benefit of the Jelly view.